static/app/components/events/eventEntry.tsx

```diff
@@ -1,5 +1,5 @@
 import Breadcrumbs from 'sentry/components/events/interfaces/breadcrumbs';
-import Csp from 'sentry/components/events/interfaces/csp';
+import {Csp} from 'sentry/components/events/interfaces/csp';
```

Comment:
i'm not sure do we respect something like https://github.com/airbnb/javascript#naming--Acronyms-and-Initialisms, but you can review it.

Reply:
sure... we can address this afterward... I just wanted to make it named export because its better :)

Reply:
👍